src/api.js

```diff
@@ -432,7 +436,7 @@ const utxoSumForAddresses: HandlerFunction = async function (req, _res) {
     if (balance.kind === 'error') {
       return {status: 400, body: balance.errMsg};
     }
-    sum = sum.plus(balance.value);
+    sum = sum.plus(String(balance.value));
```

Comment:
This is a Bignumber library call `.plus` - so updated the argument to be converted to string, because the argument might be number or bigint now, not just number. The Bignumber API processes strings or numbers similarly